# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsFrameLoader.h
# Commit: 9a255864f75d
# Full Hash: 9a255864f75ddcf4096b6222d016a914f5a43c8a
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2019-05-23 04:41:59
# Regressor Bug: 1525720
# File Overlap Count: 1
# Description:
#   Bug 1525720, part 3 - Remove method to createRemoteFrameLoader from nsIMozBrowserFrame interface. r=farre
#   
#   This appears unused and adds unneeded surface area for these API's to support.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31431
# ==============================================================================

diff -r 5f1c1b609ec1 -r 9a255864f75d dom/base/nsFrameLoader.h
--- a/dom/base/nsFrameLoader.h	Wed Apr 24 23:05:40 2019 -0500
+++ b/dom/base/nsFrameLoader.h	Wed Apr 24 23:25:56 2019 -0500
@@ -342,7 +342,7 @@
    * this object, which means you can't have called ShowRemoteFrame()
    * or ReallyStartLoading().
    */
-  void SetRemoteBrowser(nsIRemoteTab* aBrowserParent);
+  void InitializeFromBrowserParent(BrowserParent* aBrowserParent);
 
   /**
    * Stashes a detached nsIFrame on the frame loader. We do this when we're